package.json

```diff
@@ -270,7 +269,8 @@
     "precck-test": "yarn run build-local",
     "prefeature-test": "yarn run build-local",
     "prepublishOnly": "rm -rf lib && npm run build-local",
-    "test-coverage": "yarn run lint && ./scripts/test-coverage.sh",
+    "pretest-coverage": "yarn build-local",
+    "test-coverage": "nyc --silent mocha 'src/**/*_spec.ts' 'compatibility/**/*_spec.ts' && nyc --silent --no-clean node ./bin/cucumber-js && nyc report --reporter=lcov",
```

Comment:
This seemed to fit reasonably as a package script now, but happy to break it down or flip back to shell if preferred.